### PR TITLE
add calculateNumbers and its tests

### PR DESCRIPTION
**Pull request by carlosmc23**

### CODE REVIEW

**Comment by marcos-x86** [comment]
Please consider solving the static code analysis checks, according to the console output, most of them are related with code format issues. In order to solve them you can use the auto-format feature of IntelliJ IDEA, open your source code file and use the hotkey `Ctrl+Alt+L` to format the code.

Don't forget execute static code analysis tasks (before upload your code) with the following command in your environment and check the console output looking for errors:
**Microsoft Windows:**
`gradlew.bat clean check`
**GNU/Linux:**
`./gradlew clean check`

File: src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CalculateNumbers {
+
+    public List<Integer> colectNumbers(String numbers) {
+        List<Integer> numbersList = new ArrayList<>();
+        String[] splitNumbers = numbers.split(" ");
+        for (int i=0; i<splitNumbers.length; i++){
```
Comment (marcos-x86) [inline src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java:12]: Please consider using a `foreach` statement. Something like:
```java
for(String number : splitNumbers) {
  numberList.add(Integer.parseInt(number));
}
```

File: src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        }
+        return numbersList;
+    }
+
+    public String highestAndLowest(String numbers) {
+       if (numbers.length()<=1){
+           return numbers;
+       }
+       List<Integer> numbersList = colectNumbers(numbers);
+       Collections.sort(numbersList);
```
Comment (marcos-x86) [inline src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java:23]: Sorting operations are too expensive to compute with large collections, so please consider using `Collections.max()` and `Collections.min()` utility methods.

**Review by marcos-x86 — CHANGES_REQUESTED** [review]

File: src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+       int highest= numbersList.get(numbersList.size()-1);
+       int lowest= numbersList.get(0);
+       return highest+" "+lowest;
```
Comment (marcos-x86) [inline src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java:26]: Please consider avoid using `+` operator to concatenate Strings. You can use `String.join` utility method.

**Review by marcos-x86 — CHANGES_REQUESTED** [review]

File: src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java (hunk 1 above)
Comment (carlosmc23) [reply src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java:12]: ok, I apply it now thanks

File: src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java (hunk 2 above)
Comment (carlosmc23) [reply src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java:23]: Ok, thanks

File: src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java (hunk 3 above)
Comment (carlosmc23) [reply src/main/java/org/fundacionjala/coding/carlos/highestandlowest/CalculateNumbers.java:26]: ok. its done thanks

**Pull request closed without merging**